experimental/framework/frs-client/src/frsClient.ts

```diff
@@ -0,0 +1,157 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Container, Loader } from "@fluidframework/container-loader";
+import {
+    ContainerSchema,
+    DOProviderContainerRuntimeFactory,
+    FluidContainer,
+    RootDataObject,
+} from "@fluid-experimental/fluid-static";
+
+import { IRuntimeFactory } from "@fluidframework/container-definitions";
+import { RouterliciousService } from "@fluid-experimental/get-container";
+
+import {
+    FRSContainerConfig,
+} from "./interfaces";
+
+/**
+ * FRSClientInstance provides the ability to have a Fluid object backed by a FRS service
+ */
+export class FRSClientInstance {
+    constructor(private readonly serviceConnectionConfig: RouterliciousService) {
+    }
+
+    public async createContainer(
+        serviceContainerConfig: FRSContainerConfig,
+        containerSchema: ContainerSchema,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const container = await this.getContainerCore(
+            serviceContainerConfig,
+            runtimeFactory,
+            true,
+        );
+
+        const rootDataObject = await this.getRootDataObject(container);
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    public async getContainer(
```

Comment:
Can you please refactor code and reduce duplication by having createContainer & getContainer call helper function that takes extra boolean flag (create vs. load)?